samples/aspnetcore/blazor/FlightFinder/FlightFinder.Client/Components/Search.razor

```diff
@@ -5,7 +5,7 @@
         <div class="col pr-0">
             <div class="form-control d-flex">
                 <div><i>✈</i>&nbsp;&nbsp;From:</div>
-                <input list="airports" placeholder="..." bind=criteria.FromAirport />
+                <input list="airports" placeholder="..." @bind="criteria.FromAirport" />
```

Comment:
Does this work? I thought we needed the following for it to be highlighted properly in the tooling:

```suggestion
                <input list="airports" placeholder="..." @bind="@criteria.FromAirport" />
```

I believe there's work in progress currently to eliminate the need for this extra `@`.

Reply:
Same with other `@bind` usages below.

Reply:
Yes, this syntax works with bind. It doesn't work with event handlers.